content/tutorials/manage-the-rippled-server/troubleshooting/diagnosing-problems.md

```diff
@@ -0,0 +1,74 @@
+# Diagnosing Problems with rippled
+
+If you have having problems with `rippled`, the first step is to collect more information to accurately characterize the problem. From there, it can be easier to figure out a root cause and a fix.
+
+If your server does not start, see [rippled Server Won't Start](server-wont-start.html) for a list of some possible causes and fixes.
```

Comment:
It might be nice to expand a bit on what you mean by "does not start".  This covers catastrophic errors where the server dies and returns to the command prompt.  Some folks think that "does not start" includes does not sync.